_posts/2017-03-29-municipal-boundaries-update-for-first-quarter-calendar-year.md

```diff
@@ -0,0 +1,46 @@
+---
+layout: post
+status: publish
+published: true
+title: 'Municipal Boundaries update for first quarter (Calendar year)'
+author:
+  display_name: MHeagin
+  email: mheagin@utah.gov
+date: 2017-03-29 16:57:20
+categories:
+- Featured
+tags:
+- Municipalities
+- Municipal Boundaries
+- 
+
+---
+
+Municipal Boundaries update for first quarter 2017 (calendar year)
+
+Wow   Already three months in the new year, it seems to be moving along fast.
```

Comment:
throw in a "!" .   Never sure if those different characters will cause an issue with the coding